themes/bobby/bobby.theme.bash

```diff
@@ -14,7 +14,7 @@ RVM_THEME_PROMPT_SUFFIX="|"
 
 function prompt_command() {
     #PS1="${bold_cyan}$(scm_char)${green}$(scm_prompt_info)${purple}$(ruby_version_prompt) ${yellow}\h ${reset_color}in ${green}\w ${reset_color}\n${green}→${reset_color} "
-    PS1="\n$(battery_char) $(clock_char) ${yellow}$(ruby_version_prompt) ${purple}\h ${reset_color}in ${green}\w\n${bold_cyan}$(scm_char)${green}$(scm_prompt_info) ${green}→${reset_color} "
+    PS1="\n$(battery_char) $(clock_char) ${yellow}$(ruby_version_prompt) ${purple}\h ${reset_color}in ${green}\w\n${bold_cyan}$(scm_char)${green}$(git_prompt_info) ${green}→${reset_color} "
```

Comment:
Not sure about this, as it changes the Bobby theme to only work with Git repos. Previously, this used the `scm_prompt_info` command, with your change, it's using the `git_prompt_info` function, which stops support for SVN or other repos.